app/scripts/modules/docker/src/image/DockerImageUtils.ts

```diff
@@ -8,7 +8,12 @@ export interface IDockerImageParts {
 export class DockerImageUtils {
   // Split the image id up into the selectable parts to feed the UI
   public static splitImageId(imageId = ''): IDockerImageParts {
-    const imageParts = imageId.split(':');
+    let imageParts: string[];
+    if (imageId.indexOf('@') > 0) {
```

Comment:
[`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) returns`-1` if the value is not found, so technically we should switch `>0` to `>-1`. Alternately, [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) can make this a lot cleaner:

```suggestion
    if (imageId.includes('@')) {
```

Reply:
Thanks! I fixed it (15a3e10)